internal/config/config.go

```diff
@@ -103,7 +103,7 @@ func Get() *IngressConfig {
 	options.SetDefault("OpenshiftBuildCommit", "notrunninginopenshift")
 	options.SetDefault("ValidTopics", "unit,announce")
 	options.SetDefault("Profile", false)
-	options.SetDefault("DenyList", []string{})
+	options.SetDefault("Black_Listed_OrgIDs", []string{})
```

Comment:
I don't want to nitpick it, but we haven't been using `_` in other vars. We can do this, but it looks odd next to everything else.

Reply:
Yeah, I noticed this as well.  I struggle with this.  The envs end up looking like `INGRESS_BLACKLISTEDORGIDS` and that's really difficult for my simple caveman brain to understand.  I think the other env vars need to be renamed to make them more readable.

The question is ...do we start with making them more readable now?  Or wait and do it later (likely never to happen)?  I dunno.  I'm going to go with this one starts the process of making them easier to read.  :)